tests/unit/test_hnsw_tiered.cpp

```diff
@@ -436,3 +436,184 @@ TYPED_TEST(HNSWTieredIndexTest, insertJobAsyncMulti) {
     // Cleanup.
     thread_pool.clear();
 }
+
+TYPED_TEST(HNSWTieredIndexTest, deleteFromHNSWBasic) {
+    // Create TieredHNSW index instance with a mock queue.
+    std::shared_ptr<VecSimAllocator> allocator = VecSimAllocator::newVecsimAllocator();
+    size_t dim = 4;
+
+    for (auto is_multi : {false, true}) {
```

Comment:
I made a macro for logging the mode, depending on this flag to be called `isMulti`. Consider at least renaming for easier fix in the future